### PR TITLE
[CHE-38] Updated README

### PR DESCRIPTION
**Pull request by JimmyTran48**
Thank you for your contribution to Code Hammers's repository! Before merging, please fill out this brief form.

### Description

Refactored and added additional content to README

### Jira Task

[CHE-38](https://codehammers.atlassian.net/jira/software/projects/CHE/boards/3?selectedIssue=CHE-38)

### Testing Instructions

Proof read it

### Checklist

Please go through each item of this checklist carefully.

- [x] (Example checked item. Don't add any spaces around the "x".)

#### All Team Members

- [x] I added a descriptive title to this PR.
- [x] I filled out the **Description**, **Jira Task**, and **Testing Instructions** sections above.
- [ ] I added or updated [Jest unit tests]for any changes to components, server-side controllers, etc.
- [ ] I ran `npm run docker-test` in my local environment to check that this PR passes all unit tests.
- [ ] I did a quick check to make sure my code changes follow the recomended style guide.

### Additional Notes, Images, etc.

(This is space for any additional information or attachments you'd like to add as part of this PR.)

[CHE-38]: https://codehammers.atlassian.net/browse/CHE-38?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,117 @@
...
-- Open a new terminal and in the **root** run **npm run docker-dev**. This will spin up the containerized backend server and a localhost render on 8080.
-- Login credentials for testing are email: tester@codehammers.com, password: ilovetesting
+As you know, alums have access to the Codesmith intranet. While, the intranet is a staple for many during the job search, it lacks a bit of flair. Thus the idea to create a robust intranet for alumni. Enter Code Hammers!
 
-You will need to set up the .env file. This should include **"NODE_ENV=development"** as well as any database connection strings, api keys, seeder phrases as required.
+## Why Contribute?
+
+Building and contributing is the best way to learn how to code! We wanted to provide the opportunity for those looking to sharpen their skills to contribute to a large codebase.
+
+1. Eperience the development workflow on a much greater scale
```
Comment (brok3turtl3) [inline README.md:11]: E***x***perience

File: README.md [hunk 2]
```diff
@@ -1,11 +1,117 @@
...
+2. Recieve code reviews and guidance on tasks
```
Comment (brok3turtl3) [inline README.md:12]: Rec***ei***ve

File: README.md [hunk 3]
```diff
@@ -1,11 +1,117 @@
...
+3. Work with engineers currently in the industry (get those referrals!)
+4. It's FUN!!!
+
+## Table of Contents
+
+1. [Dev Environment Setup](#dev-environment-setup)
+2. [Startings the Application](#starting-the-application)
```
Comment (brok3turtl3) [inline README.md:19]: Starting~~s~~ the Application...

File: README.md [hunk 4]
```diff
@@ -1,11 +1,117 @@
...
+With any large codebase, there is always going to be some sort of environment setup. Code Hammers is a [monorepo](https://en.wikipedia.org/wiki/Monorepo), the repo for the frontend code and backend code are contained within one repo. You will notice there are multiple `package.json` files. This is not always the case!
+
+- Clone the repo
+- Install packages
+- Create an `.env` file
+  - Reach out Sean or Jimmy for all secret variables
+- Run the initial tailwind script
+  - `cd` to the `./client`folder and run `npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch` in your terminal.
+  - You will be prompted to install Tailwind - Choose YES.
+  - This will run the inital css build. It will also leave it running and provide live compiles for any changes to styling.
```
Comment (brok3turtl3) [inline README.md:33]: init***i***al
~~css~~ CSS

File: README.md [hunk 5]
```diff
@@ -1,11 +1,117 @@
...
+  const {prop1, prop2} = props;
+}
+
+// worse
+const MyComponent = (props) => {
+  ...
+  props.prop1
+  props.prop2
+}
+```
```
Comment (brok3turtl3) [inline README.md:78]: Capitalize good, bad, and worse?

**Review by brok3turtl3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (JimmyTran48) [reply README.md:12]: Eventually?

**Review by brok3turtl3 — APPROVED** [review]

**Pull request merged**